shadowed/antlr/src/main/antlr/com/autonomousapps/internal/grammar/gen/GradleGroovyScriptLexer.java

```diff
@@ -1,4 +1,4 @@
-// Generated from /Users/trobalik/Development/dependency-analysis-android-gradle-plugin/shadowed/antlr/src/main/antlr/com/autonomousapps/internal/grammar/GradleGroovyScriptLexer.g4 by ANTLR 4.10.1
+// Generated from /Users/joshfeinberg/projects/dependency-analysis-android-gradle-plugin/shadowed/antlr/src/main/antlr/com/autonomousapps/internal/grammar/GradleGroovyScriptLexer.g4 by ANTLR 4.10.1
```

Comment:
Do we want to not commit this generated code? Not sure if it makes sense to add this to .gitignore

Reply:
We absolutely must commit this generated code. It must go into the compiled jar that is published on maven central.